src/init.cpp

```diff
@@ -670,6 +673,17 @@ static void CleanupBlockRevFiles()
     }
 }
 
+#if HAVE_SYSTEM
+static void StartupNotify()
+{
+    std::string strCmd = gArgs.GetArg("-startupnotify", "");
```

Comment:
style-nit: Could pass in the argsmanager to avoid referencing the global?

```suggestion
    std::string cmd = args.GetArg("-startupnotify", "");
```

Reply:
Done